docs/lesson-plans/week-1.md

```diff
@@ -0,0 +1,726 @@
+get errors. 
+- For the backend, make sure that you set the `JAVA_HOME` variable and that everything else required
+is installed.
+    - `JAVA_HOME` is a system-level variable which lets programs know where to look for the JDK (Java Development Kit). 
+- For Git bash, the terminal may need to be manually set in every project. Don't forget to do that, or
+else you'll end up running commands that don't exist in Windows Command Prompt.
+    - Git bash is a program which lets you run a Mac/Linux like terminal on Windows. We describe it  little more below.
+
+## Terminal Basics
+
+The terminal is a really important tool, which you'll find extremely useful throughout your 
+developer career. Not only does it allow you to access settings and commands which you would otherwise
+not have access to, it sometimes is just easier to be able to do some things by typing out commands
+rather than pointing and clicking with a mouse. 
+
+Learning how to use the terminal well may take a while, but we'll get you started with a few commands
+we think every developer should know. We'll also show you how to build your projects if you 
+haven't tried doing that yet.
+
+>Note: we'll be showing you how to use unix-based bash commands, which means that these commands will 
+>work for Mac and Linux computers. If you have a Windows computer, either download the Linux subsystem
+>or follow the commands in the backend README.md to install Git bash and start using that in your 
+>IntelliJ projects.
+
+### Common Commands
+
+The terminal is a tool which uses specific commands to perform system-level actions. The way that it works
+is each terminal instance (tab/window) works in a specific directory, and a lot of the commands utilize the
+files in the given directory or a subdirectory. The actual terminal syntax depends on which 'shell', or 
+command language/interface, is being used. The Windows operating system uses the Command Prompt as a shell,
+while Mac and Linux often use BASH (Bourne Again SHell) as a shell.
+
+If you want to learn more about the terminal, check out [this video](https://www.youtube.com/watch?v=oxuRxtrO2Ag).
+It's a bit long, but it should give you a pretty solid understanding of how to use the terminal and some
+common commands.
+
+#### cd
+
+`cd` is arguable one of the most used commands in the terminal. It stands for 'change directory', and it
+allows you to navigate around your computer. 
+
+Format:
+```
+cd <directory>
+```
+
+- You can chain multiple directories together by adding '/' between each directory if you would like
+to jump to a specific one.
+- You can jump up directories with the '..' notation.
+- You can jump to your home directory (the one with Desktop, Documents, ... inside of it) by just typing `cd` or
+`cd ~/`
+
+Examples:
+```
+# Go to your desktop
+cd Desktop
+
+# Go to back to your home directory (the one where all your stuff is)
+cd ..
+
+# Go up multiple directories
+cd ../../../
+
+# Go back to your home directory
+cd ~/
```

Comment:
I've never added a `/` after `~`. I think it's weird to have it here

Reply:
I use it a lot :(. It's really useful if you're working outside of home but need to reference files there.